tests/std/tests/VSO_0157762_feature_test_macros/test.cpp

```diff
@@ -1073,19 +1073,29 @@ STATIC_ASSERT(__cpp_lib_apply == 201603L);
 #endif
 #endif
 
-#if CXX17_MODE
+
+#if CXX20_MODE

```

Comment:
I filed https://github.com/microsoft/STL/issues/600 to track improvements to this test. For now, altering this preprocessor chunk is fine (although technically it's in "the wrong section" after your change). No change requested.